scripts/_env_var_util.sh

```diff
@@ -89,12 +121,20 @@ _env_var_util::set_defaults () {
     METADATA="${METADATA:-"{}"}"
     ACCELERATOR_TYPE=${ACCELERATOR_TYPE:-"nvidia-tesla-a100"}
     LABELS="${LABELS:-"{}"}"
-    [ -n "${IMAGE_NAME}" ] || {
-        IMAGE_FAMILY_NAME=${IMAGE_FAMILY_NAME:-"pytorch-1-12-gpu-debian-10"};
+    [ -z "${IMAGE_FAMILY_NAME}" ] && [ -z "${IMAGE_NAME}" ] && {
+        if [ "${ORCHESTRATOR_TYPE}" = "slurm" ]; then
+            IMAGE_FAMILY_NAME='schedmd-v5-slurm-22-05-6-hpc-centos-7'
+            IMAGE_PROJECT='schedmd-slurm-public'
+        else
+            IMAGE_FAMILY_NAME='pytorch-1-12-gpu-debian-10'
+        fi
     }
+    IMAGE_PROJECT=${IMAGE_PROJECT:-"ml-images"}
```

Comment:
This is a one-liner for setting a default value to a parameter.

from [the bash manual](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html):
> `${parameter:-word}`
>
> If `parameter` is unset or null, the expansion of `word` is substituted. Otherwise, the value of `parameter` is substituted.